src/meshapi/validation.py

```diff
@@ -29,15 +30,22 @@ def validate_email_address(email_address: str) -> bool:
     )
 
 
+def normalize_phone_number(phone_number: str) -> str:
+    return phonenumbers.format_number(
+        phonenumbers.parse(phone_number, "US"),
+        phonenumbers.PhoneNumberFormat.INTERNATIONAL,
+    )
+
+
 # Expects country code!!!!
```

Comment:
Is this line true anymore? Can I just pass `555-555-5555` or `555 555 5555` and have it default to US?

Reply:
```suggestion
```

Reply:
Make sure to add a test for interaction with different country codes.

Reply:
Yes, we can pass plain US phone numbers now. Added tests to validate this (and that international phone numbers parse fine)

Reply:
Though I still think maybe we should require +1 on the join form? There's probably a nice little widget we can use to show a little country flag or something and do a bit of validation, etc